File: Data Structures & Algorithms/maximum-subarray/submission-2.cs
public class Solution {
    public int MaxSubArray(int[] nums) {
        int maxsub = nums[0];
        int currsum = 0;

        foreach(int num in nums)
        {
            if(currsum < 0)
            {
                currsum = 0;
            }

            currsum += num;
            maxsub = Math.Max(maxsub, currsum);
        }
        return maxsub;
    }
}

File: Data Structures & Algorithms/subsets/submission-2.py
class Solution:
    def subsets(self, nums: List[int]) -> List[List[int]]:
        res = []
        res.append([])
        #res+=[[]]

        for num in nums:
            size = len(res)
            for i in range(size):
                res+= [[num] + res[i]]
                #res.append([num] + res[i])
        return res